contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -1620,6 +1619,7 @@ sp_describe_undeclared_parameters_internal(PG_FUNCTION_ARGS)
 	}
 	else
 	{
+		is_supported_case_sp_describe_undeclared_parameters = true;
```

Comment:
How are we making sure that we don't leak the value of this static variable in case of an error? I don't see a try/catch block which would prevent this so be careful.

Reply:
If we put the code block inside a TRY/CATCH block then we won't get some desired ERROR message hence did not take that approach. Rather set the value of is_supported_case_sp_describe_undeclared_parameters in the SRF_IS_FIRSTCALL for any new query. This means, even if something goes wrong, for the next query the parameter's value will be set again to true at the very beginning. So I think this should be able to handle any such abrupt error cases.

Reply:
Added TRY/CATCH block in latest rev.